common/utils/capitalise.js

```diff
@@ -0,0 +1,7 @@
+// @flow
+export default (word: string): string => {
+  const [firstLetter] = word;
+  const restOfWord = word.slice(1);
+
+  return `${firstLetter.toUpperCase()}${restOfWord}`;
+};
```

Comment:
Is there not a grammar file we can put all of these into?
Don't really mind, just a thought.

Reply:
yeah, I wondered the same – `grammar.js`?

Reply:
Next question – `capitalise/camelise` or `capitalize/camelize`?